yarn.lock

```diff
@@ -3363,11 +3377,6 @@ eslint-scope@^4.0.0, eslint-scope@^4.0.3:
     esrecurse "^4.1.0"
     estraverse "^4.1.1"
 
-esm@~3.0.84:
```

Comment:
don;t we need esm anymore?

Reply:
it was not a direct dependency in package.json. May be an internal dependency of  angular-plotly package. so removed now.